CHANGELOG.rst

```diff
@@ -2,6 +2,94 @@
+    - Added copy button to the modal, which allows the user to copy the configuration used by the Form's
+      registration backend. This is shown only when backends are available.
+    - Added dropdown to select a registration backend instead of always picking the first one.
+* [:backend:`4745`] Added registration variable containing all attachment URLs.
+* [:backend:`4796`] Added product url to ZGW APIs registration.
+* [:backend:`4762`] Added cosign data (bsn) to the StUF-ZDS creerzaak bericht.
+* [:backend:`4798`] Added custom confirmation modal, which replaces the old way of displaying confirmation
+  messages.
+* [:backend:`4344`] It's now possible to configure Objects API group in ZGW registration. This used to be by
+  default the one with the lowest primary key.
+* [:backend:`4320`] Cosign improvements. 
+    - Updated translations so the suggested texts are already included.
+    - Added the ability to add a 'cosign now' button on confirmation page.
+    - Made the cosign verification flow faster when the code is included in the link in the email.
+    - Made the confirmation page title and body configurable.
+    - Updated cosign configuration docs and manual.
+* [:backend:`4815`] Changed submission removal limit to 0, allowing submissions to be deleted after 0 days
+  (i.e. on the same day).
+
+* [:backend:`4717`] Improved accessibility for site logo, error message element and PDF documents. 
+* Replaced duplicated code for payment/registration plugin configuration option forms, by adding a generic
+  component.
+* Refactored demo plugins configuration to use a modal instead of inline configuration options.
+* [:backend:`4707`] Added the option to resize the json widget manually.    
+* [:backend:`4686`] Improved UX for email and StUF-ZDS registrations by using React components, instead of
+  react-json-schema-form generated forms.
+* [:backend:`4720`] Improved accessibility for the skiplink and the PDF report.
+* [:backend:`4396`] Added Objects API prefill plugin.
+
+**Bugfixes**
+
+* Fixed tests after removing password component.
+* [:backend:`4732`] Fixed CSP issues for Expoints/other analytics.
+* Fixed the logic rules manual regarding "duration" which did not work.
+* [:backend:`4823`] Fixed uploading files with leading or trailing whitespace in the
+  filename.
+* [:backend:`4810`] Fixed uppercase component variable values turing lowercase. See the
+  remark above for additional instructions.
+* Fixed recursion issues in component search strategies.
+* [:backend:`4826`] Added a work around to fix the translation issue and to properly evaluate the
+  translated default values for the newly added configuration model fields.
+* [:backend:`4772`] Fixed Select components' unset dataType which caused issues when submitting data to
+  the backend.
+* [:backend:`4727`] Fixed crash when a user defined variable was changed to an array
+  datatype.
+* [:backend:`4719`] Improved accessibility in postcode fields.
```

Comment:
Bugfix? or new feature? :thinking: "Improved" reads as a new feature, otherwise it should be "Fixed accesibility..."